src/command/backup/backup.c

```diff
@@ -2574,14 +2690,88 @@ cmdBackup(void)
         const time_t timestampStart = backupTime(backupData, false);
 
         // Check if there is a prior manifest when backup type is diff/incr
-        Manifest *const manifestPrior = backupBuildIncrPrior(infoBackup);
+        Manifest *manifestPrior = backupBuildIncrPrior(infoBackup);
+
+        // Perform preliminary copy of full/incr backup
+        uint64_t copySizePrelim = 0;
+
+        if (cfgOptionStrId(cfgOptType) == backupTypeFull && cfgOptionBool(cfgOptBackupFullIncr))
+        {
+            ASSERT(manifestPrior == NULL);
+
+            MEM_CONTEXT_TEMP_BEGIN()
+            {
+                // Build the manifest
+                Manifest *const manifestPrelim = manifestNewBuild(
+                    backupData->storagePrimary, infoPg.version, infoPg.catalogVersion, timestampStart,
+                    cfgOptionBool(cfgOptOnline), cfgOptionBool(cfgOptChecksumPage), cfgOptionBool(cfgOptRepoBundle),
+                    cfgOptionBool(cfgOptRepoBlock), &blockIncrMap, strLstNewVarLst(cfgOptionLst(cfgOptExclude)),
+                    dbTablespaceList(backupData->dbPrimary));
+
+                // Calculate the expected size of the final copy
+                uint64_t copySizeFinal = backupManifestCopySize(manifestPrelim);
+
+                // Remove files that do not need to be considered for the preliminary copy
```

Comment:
Updated to:
```
                // Remove files that do not need to be considered for the preliminary copy because they were modified after the
                // calculated limit time and are therefore likely to be modified during the backup
```